.changeset/five-frogs-heal.md

```diff
@@ -0,0 +1,6 @@
+---
+"@vygruppen/spor-react": minor
+"@vygruppen/spor-icon": minor
```

Comment:
Du må legge til changeset for både spor-icon-react og spor-icon-react-native. spor-icon er bare en samlepakke for disse, og du vil få en del versjoneringsfeil uten å oppdatere de. De må velges manuelt.

En guide til icons kan du finne her: https://spor.vy.no/guides/how-to-add-new-icons